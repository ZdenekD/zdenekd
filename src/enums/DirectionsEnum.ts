enum DirectionsEnum {
    up = 'up',
    right = 'right',
    down = 'down',
    left = 'left'
}

export default DirectionsEnum;
