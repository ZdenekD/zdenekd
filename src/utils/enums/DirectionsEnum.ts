export enum DirectionsEnum {
    up = 'up',
    right = 'right',
    down = 'down',
    left = 'left'
}
