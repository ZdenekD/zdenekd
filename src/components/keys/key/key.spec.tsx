import {render, screen} from '@testing-library/react';
import DirectionsEnum from '@/enums/DirectionsEnum';
import Key from '.';

describe('Keys/Key', () => {
    it('renders without error', () => {
        render(
            <Key
                direction={DirectionsEnum.down}
                label="Down"
            />
        );

        expect(screen.getByTestId('component-key')).toBeInTheDocument();
    });
});
