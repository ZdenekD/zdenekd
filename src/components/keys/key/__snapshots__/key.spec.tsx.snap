// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Key match snapshot 1`] = `
<button type="button"
        class="key "
        aria-label="Down"
        data-test="component-key"
>
  <i class="icon down">
  </i>
</button>
`;
